packages/core-sdk/resolvers/account-resolvers/interface/query.graphql

```diff
@@ -6,11 +6,10 @@ type Query {
   getTokenTransfers(
     accountAddress: String!
     tokenAddress: String!
-    vsCurrency: String!
```

Comment:
Did you find that vsCurrency isn't necessary? I'm just curious. I don't know much about the API.